packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -67,7 +68,11 @@ export const Workbench: React.FunctionComponent<{
   const activeAction = model ? highlightedAction || selectedAction : undefined;
   const [selectedTime, setSelectedTime] = React.useState<Boundaries | undefined>();
   const [sidebarLocation, setSidebarLocation] = useSetting<'bottom' | 'right'>('propertiesSidebarLocation', 'bottom');
-  const [showRouteActions, , showRouteActionsSetting] = useSetting('show-route-actions', true, 'Show route actions');
+  const [, , showRouteActionsSettingInternal] = useSetting(showRouteActionsSetting ? '' : 'show-route-actions', true, 'Show route actions');
```

Comment:
```suggestion
  const [, , showRouteActionsSettingInternal] = useSetting(showRouteActionsSetting ? undefined : 'show-route-actions', true, 'Show route actions');
```

The type allows passing `undefined`, so let's use that.